README.md

```diff
@@ -394,9 +444,9 @@ ResultMetaData result=ImageKit.getInstance().getRemoteFileMetadata(url);
 System.out.println("======FINAL RESULT=======");
 System.out.println(result);
 System.out.println("Raw Response:");
-System.out.println(result.getRaw());
+System.out.println(result.getResponseMetaData().getRaw());
 System.out.println("Map Response:");
-System.out.println(result.getMap());
+System.out.println(result.getResponseMetaData().getMap());
 ```
 
```

Comment:
The order of APIs in readme.md should match that of checklit in Notion document or what is there on docs.imagekit.io. Currently it seems that readme.md was updated as soon as that API implementation was done and it is all mixed up and out of any order and structure.